### PR TITLE
Bump postcss to 8.4.5 for react samples

### PR DESCRIPTION
**Pull request by cristobal-buenrostro**

### CODE REVIEW

File: js/samples/react-sample-server/client/yarn.lock [hunk 1]
```diff
@@ -8820,6 +8830,15 @@ postcss@^8.1.0:
     picocolors "^1.0.0"
     source-map-js "^0.6.2"
 
+postcss@^8.4.5:
+  version "8.4.5"
+  resolved "https://registry.yarnpkg.com/postcss/-/postcss-8.4.5.tgz#bae665764dfd4c6fcc24dc0fdf7e7aa00cc77f95"
+  integrity sha512-jBDboWM8qpaqwkMwItqTQTiFikhs/67OYVvblFFTM7MrZjt6yMKd6r2kgXizEbTTljacm4NldIlZnhbjr84QYg==
+  dependencies:
+    nanoid "^3.1.30"
```
Comment (rwallerms) [inline js/samples/react-sample-server/client/yarn.lock:8838]: nanoid just got updated to 3.2.0

File: js/samples/react-sample-simple/yarn.lock [hunk 2]
```diff
@@ -8825,6 +8830,15 @@ postcss@^8.1.0:
     picocolors "^1.0.0"
     source-map-js "^0.6.2"
 
+postcss@^8.4.5:
+  version "8.4.5"
+  resolved "https://registry.yarnpkg.com/postcss/-/postcss-8.4.5.tgz#bae665764dfd4c6fcc24dc0fdf7e7aa00cc77f95"
+  integrity sha512-jBDboWM8qpaqwkMwItqTQTiFikhs/67OYVvblFFTM7MrZjt6yMKd6r2kgXizEbTTljacm4NldIlZnhbjr84QYg==
+  dependencies:
+    nanoid "^3.1.30"
```
Comment (rwallerms) [inline js/samples/react-sample-simple/yarn.lock:8838]: 3.2.0

**Review by rwallerms — CHANGES_REQUESTED** [review]

File: js/samples/react-sample-server/client/yarn.lock (hunk 1 above)
Comment (cristobal-buenrostro) [reply js/samples/react-sample-server/client/yarn.lock]: correct, I didn't have that change in my branch, updated now.

File: js/samples/react-sample-simple/yarn.lock (hunk 2 above)
Comment (cristobal-buenrostro) [reply js/samples/react-sample-simple/yarn.lock]: done

**Review by rwallerms — APPROVED** [review]

**Pull request merged**